python/tskit/trees.py

```diff
@@ -5146,7 +5146,13 @@ def variants(
             isolated_as_missing=isolated_as_missing,
             alleles=alleles,
         )
-        start, stop = np.searchsorted(self.tables.sites.position, interval)
+        if left == 0 and right == self.sequence_length:
+            # Special case: no need to search the whole of the the positions array
```

Comment:
typo. comment seems unnecessary, anyhow.

Reply:
Changed, and a few similar typos in the file corrected.